stdlib/mexpr/pprint.mc

```diff
@@ -208,7 +208,7 @@ lang AppPrettyPrint = PrettyPrint + AppAst
     else error "Impossible"
 end
 
-lang FunPrettyPrint = PrettyPrint + FunAst
+lang FunPrettyPrint = PrettyPrint + FunAst + UnknownTypeAst
```

Comment:
I suppose splitting out `Unknown` has implications here. You could make one fragment that does printing in the presence of unknown and one in the absence, the former would have a more specific pattern, so they would merge correctly.

Another option is of course also to not special case unknown, print it like any other type, though that depends on it being possible to explicitly write the type `Unknown` in the file, otherwise the pretty printed version wouldn't parse the same.

Reply:
I think a version of your second suggestion makes the most sense: print the type `Unknown` as `""` (empty string) in `getTypeStringCode` (or, I guess the more appropriate thing is to return an `Option String` from `getTypeStringCode`), and then detect this in `pprintCode` (to remove the `:` as well)